index.md

```diff
@@ -1,5 +1,5 @@
 ### About
-This library is featuring linear algebra solvers for the solution of spatially and temporally discretized mechanics problems. In its core, it serves as a binding between the [LinearAlgebra](https://github.com/mgroupntua/linearalgebra) and the [MSolve.Core](https://github.com/mgroupntua/MSolve.Core) repos.
+This library is featuring nonlinear and dynamic analyzers for the solution of spatially and temporally discretized mechanics problems. In its core, it serves as a binding between the [LinearAlgebra](https://github.com/mgroupntua/linearalgebra) and the [MSolve.Core](https://github.com/mgroupntua/MSolve.Core) repos.
```

Comment:
The part "In its core, it serves as a binding between the [LinearAlgebra](https://github.com/mgroupntua/linearalgebra) and the [MSolve.Core](https://github.com/mgroupntua/MSolve.Core) repos." should probably be removed